server/game_components/game.py

```diff
@@ -105,6 +106,29 @@ def get_player(self, _uid: int) -> Player | None:
     def get_mob(self, _uid: int) -> Mob | None:
         return self.mobs.get(_uid)
 
+    def spawn_mobs(self) -> None:
+
+        for room in self.rooms.values():
+            if room.get_map_location() == (15, 23):
+                m = Mob("Mite", ["nibble", "eat_berry", "stomp", "annoy"], self)
+                # print(f"Adding mite at {room.get_map_location()}")
+                room.add_mob(m)
+                continue
+            if isinstance(room, Wall):
+                continue
+            if isinstance(room, SpidersDen):
+                m = Mob("Spider", ["sting", "eat_berry", "nibble"], self)
+                # print(f"Adding spider at {room.get_map_location()}")
+                room.add_mob(m)
+                continue
+            chance_to_spawn = random.randint(0, 100)
+
+            # if there are too many mobs, make this magic number lower
+            if chance_to_spawn < 25:
```

Comment:
Maybe we could make it so that no mobs can spawn in the spawn location?

Reply:
oh good call, ill fix